tests/storage-volumes-vm

```diff
@@ -254,6 +254,10 @@ do
 		lxc storage volume attach "${poolName}" virtual-machine/v2 v1 v2-root
 		sleep 3
 		lxc exec v1 -- test -L /dev/disk/by-id/scsi-0QEMU_QEMU_HARDDISK_lxd_v2--root
+
+		# Test that unmount succeeds; failed unmount will not cause VM stop to fail.
```

Comment:
If I followed correctly, this detach and reattach the same disk (`v2`'s root) to `v1`. If so, I don't understand the comment about a failed umount not impacting the VM stop.

Is it that `detach` will succeed even if `umount` fails from the instance? Or is it that a normal (non-force) `lxc stop v1` would work?

Reply:
`lxc stop` (with or without `-f`) won't fail if one of it's devices fails to stop.

Other way around: `detach` will fail if the `unmount` fails.

I pushed a clarification to the comment (hopefully).